src/app/pages/HomePage/components/BlogPart/BlogPart.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+import "./BlogPart.scss";
+import vector from "../../../../../assets/icons/vectorImage.svg";
+
+interface BlogPartProps {
+  date: string;
+  tag: string;
+  imageSrc: string;
+  content: string;
+}
+
+export const BlogPart = (props: BlogPartProps) => {
```

Comment:
you're using word "article" so often in your code...
maybe it's better to rename this component to BlogArticle?)